src/main.cpp

```diff
@@ -59,22 +59,29 @@ std::string make_response(const json& response)
     return header + content.dump(4);
 }
 
+static bool ends_with(const std::string haystack, const std::string needle) {
+    if (needle.length() > haystack.length())
+        return false;
+    return haystack.compare(haystack.length() - needle.length(), needle.length(), needle) == 0;
+}
+
```

Comment:
Wouldn't `string::ends_with` or `ranges::ends_with` be fine here, too?

Reply:
I didn't see it was added in C++20 - that means bumping `CMAKE_CXX_STANDARD` to 20 though, is that alright?

Reply:
Yeah, that's fine.